playbooks/ip_partnership/README.txt

```diff
@@ -0,0 +1,43 @@
+Objective:
+Configure IP Partnership and create multiple Remote Copy replationships along with volumes in an easy way.
+
+Prerequisite:
+- IBM storage Virtualize ansible collection plugins must be installed
+
+These playbooks creates IP partnership and Remote copy relationships between a primary cluster and the secondary cluster.
+  - It uses storage virtualize ansible modules.
+  - These playbooks are designed to create IP partnership between source cluster to destination cluster. These are designed in a way that it assigns the IP paddresses if specified and uses if existing IPs if there are only portsets details passed in inventroy file
```

Comment:
paddresses -> addresses

Reply:
Can we rephrase this sentence - "These are designed in a way that it assigns the IP paddresses if specified and uses if existing IPs if there are only portsets details passed in inventroy file"